pythonforandroid/bootstraps/common/build/templates/Service.tmpl.java

```diff
@@ -19,29 +19,44 @@ protected int getServiceId() {
     }
 
     static public void start(Context ctx, String pythonServiceArgument) {
-        Intent intent = getDefaultIntent(ctx, pythonServiceArgument);
+        Intent intent = getDefaultIntent(ctx, "", "{{ args.name }}",
+					 "{{ name|capitalize }}",
+					 pythonServiceArgument);
         ctx.startService(intent);
```

Comment:
What about calling `start(ctx, "", "{{ args.name}}", "{{ name|capitalize }}", pythonServiceArgument)` instead of keeping duplicated code?